migrations/2018_01_18_134400_change_password_tokens_id_to_token.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+
+return Migration::renameColumn('password_tokens', 'id', 'token');
```

Comment:
For consistency, this migration file should be named change_password_tokens_**rename**_id_to_token.php